Tests/test_file_ppm.py

```diff
@@ -100,6 +105,44 @@ def test_pnm(tmp_path):
         assert_image_equal_tofile(im, f)
 
 
+def test_pfm(tmp_path):
+    with Image.open("Tests/images/hopper.pfm") as im:
+        assert im.info["scale"] == 1.0
+        assert_image_equal(im, hopper("F"))
+
+        f = str(tmp_path / "tmp.pfm")
+        im.save(f)
+
+        assert_image_equal_tofile(im, f)
+
+
+def test_pfm_big_endian(tmp_path):
+    with Image.open("Tests/images/hopper_be.pfm") as im:
+        assert im.info["scale"] == 2.5
+        assert_image_equal(im, hopper("F"))
+
+        f = str(tmp_path / "tmp.pfm")
+        im.save(f)
+
+        assert_image_equal_tofile(im, f)
```

Comment:
Let's rename `f` to `filename`:
```suggestion
        filename = str(tmp_path / "tmp.pfm")
        im.save(filename)

        assert_image_equal_tofile(im, filename)


def test_pfm_big_endian(tmp_path):
    with Image.open("Tests/images/hopper_be.pfm") as im:
        assert im.info["scale"] == 2.5
        assert_image_equal(im, hopper("F"))

        filename = str(tmp_path / "tmp.pfm")
        im.save(filename)

        assert_image_equal_tofile(im, filename)
```